tests/core/test_config.py

```diff
@@ -110,3 +112,126 @@ def testDefaultChecks(self):
 
         for c in DEFAULT_CHECKS:
             self.assertTrue(c in init_checks_names)
+
+
+TEMP_3RD_PARTY_CHECKS_DIR = '/tmp/dd-agent-tests/3rd-party'
+TEMP_ETC_CHECKS_DIR = '/tmp/dd-agent-tests/etc/checks.d'
+TEMP_ETC_CONF_DIR = '/tmp/dd-agent-tests/etc/conf.d'
+TEMP_AGENT_CHECK_DIR = '/tmp/dd-agent-tests'
+FIXTURE_PATH = 'tests/core/fixtures/checks'
```

Comment:
to be more flexible we could replace this with: `FIXTURE_PATH =  os.path.join(os.path.dirname(os.path.realpath(__file__)), 'fixtures', 'checks')`